content/en/blog/_posts/2022-12-06-kubernetes-1.25-blog.md

```diff
@@ -0,0 +1,205 @@
+
+#### CSI migration for Azure File and vSphere graduated to Stable
+
+Both the [vSphere](https://github.com/kubernetes/enhancements/issues/1491) and [Azure](https://github.com/kubernetes/enhancements/issues/1885) in-tree driver migration to CSI have graduated to Stable. You can find more information about them in the [vSphere CSI driver](https://github.com/kubernetes-sigs/vsphere-csi-driver) and [Azure File CSI driver](https://github.com/kubernetes-sigs/azurefile-csi-driver) repositories.
+
+#### In-tree GlusterFS driver removal
+
+Already deprecated in the v1.25 release, the in-tree GlusterFS driver was [removed](https://github.com/kubernetes/enhancements/issues/3446) in this release.
+
+#### In-tree OpenStack Cinder driver removal
+
+This release removed the deprecated in-tree storage integration for OpenStack (the `cinder` volume type). You should migrate to external cloud provider and CSI driver from https://github.com/kubernetes/cloud-provider-openstack instead. For more information, visit [Cinder in-tree to CSI driver migration](https://github.com/kubernetes/enhancements/issues/1489).
+
+### Signing Kubernetes release artifacts graduates to Stable
+
+Introduced in [v1.24](https://kubernetes.io/blog/2022/05/03/kubernetes-1-24-release-announcement/), this feature constitutes a significant milestone in improving the security of the Kubernetes release process. All release artifacts are signed using [cosign](https://github.com/sigstore/cosign/), and both binary artifacts and images [can be verified](https://kubernetes.io/docs/tasks/administer-cluster/verify-signed-images/).
+
+### Support for Windows privileged containers graduates to Stable
+
+Privileged container support allows containers to run with similar access to the host as processes that run on the host directly. Support for this feature in Windows nodes, called HostProcess containers, will now [graduate to Stable](https://github.com/kubernetes/enhancements/issues/1981), enabling access to host resources (including network resources) from privileged containers.
+
+### Improvements to Kubernetes metrics
+
+This release has several noteworthy improvements on metrics.
+
+#### Metrics framework extension graduates to Alpha
+
+The **metrics framework extension** graduates to Alpha and [documentation](/docs/reference/instrumentation/metrics/) is now published for every metric in the Kubernetes codebase.
+
+#### Component Service Level Indicators graduates to Alpha
```

Comment:
kep issue: https://github.com/kubernetes/enhancements/issues/3466 

maybe mentioning "Health SLIs" not just SLIs is more accurate

Reply:
Done.